source/components/molecules/PersonField/PersonField.tsx

```diff
@@ -0,0 +1,240 @@
+import React, {useContext} from 'react'
+import PropTypes from 'prop-types'
+import styled, { ThemeContext } from 'styled-components/native'
+import { Avatar } from 'react-native-elements'
+import { Text, Button } from '../../atoms'
+import { InputRow } from '../RepeaterField/RepeaterField';
+
+// TODO: BREAKOUT TO CARD COMPONENT REDUNDANT USE IN FIELDSET ALSO.
+const PersonFieldContainer = styled.View`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: auto;
+  border-radius: 9.5px;
+  overflow: hidden;
+  margin-bottom: 24px;
+  margin-top: 16px;
+  padding-bottom: 20px;
+  padding-top: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  background: ${props => props.theme.colors.complementary.blue[3]};
```

Comment:
Should the background always be blue, no matter what colorScheme we send in as prop? Same comments for the few other places where you default to blue instead of using a dynamic prop.